Puzzlepart/did
# pwa splash screen, mobile improvements

### Your checklist for this pull request

### Review checklist

### Description
* Splash screen for IOS devices
![RPReplay_Final1602073506](https://user-images.githubusercontent.com/7300548/95332081-3b2a7480-08ab-11eb-831d-1191b79e3296.gif)
* Slightly larger Mobile headers, with theme color
* Disabled IOS press-and-hold callout (i.e. "define [word]")
* Enabled `:hover` and `:active` css pseudoclasses for IOS by doing `<body ontouchstart=""` - [see here for details](https://medium.com/appscope/designing-native-like-progressive-web-apps-for-ios-1b3cdda1d0e8)


### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. Ensure you're running with a localtunnel subdomain that's registered as a callback URI for the did AAD app registration
3. Open Safari and navigate to your localtunnel did url
4. Click "[share button] -> add to home screen
5. Open the app 
6. Observe the app has a splash screen

### Related issues
N/A

### Related PRs
#588
